.github/workflows/test.yml

```diff
@@ -43,4 +43,16 @@ jobs:
         run: |
           go get -v -t -d ./...
       - name: Test
-        run: go test -race ./...
+        run: go test -race `go list ./... | grep -v integrationtest`
```

Comment:
Separate the usual tests, from the heaviest one that also spin up PG, IPFS (from PG) and devnet.

Both sets of tests are run, but in different jobs.